docs/datatypes/matrices.md

```diff
@@ -181,33 +181,30 @@ Math.js uses geometric dimensions:
 - A vector is one-dimensional.
 - A matrix is two or multidimensional.
 
-The size of a matrix can be calculated with the function `size`. Function `size`
-returns a `Matrix` or `Array`, depending on the configuration option `matrix`.
-Furthermore, matrices have a function `size` as well, which always returns
-an Array.
+The size of a matrix can be calculated with the function `size`. Function size returns an `Array`, giving the length of its input (`Matrix` or `Array`) in each dimension. You can also call `size()` as a method on a Matrix.
 
 ```js
 // get the size of a scalar
-math.size(2.4)                                // Matrix, []
-math.size(math.complex(3, 2))                 // Matrix, []
-math.size(math.unit('5.3 mm'))                // Matrix, []
+math.size(2.4)                                // []
```

Comment:
Since we are labeling most everything elsewhere on this page explicitly as Array or Matrix, please keep the Array label on the items you have touched that are arrays, rather than assuming that it's clear that no label corresponds to Array.